.github/workflows/build-solr.yml

```diff
@@ -35,8 +35,19 @@ jobs:
             packages: write
 
         steps:
-            - name: Build Image
-              uses: hathitrust/github_actions/build@v1.4.0
+            - name: Build Solr
+              uses: hathitrust/github_actions/build@v1
+              with:
+                image: ghcr.io/hathitrust/catalog-solr
+                dockerfile: example-index/Dockerfile
+                img_tag: ${{ inputs.img_tag }}
+                tag: ${{ inputs.ref }}
+                push_latest: ${{ inputs.push_latest}}
+                registry_token: ${{ github.token }}
+                rebuild: ${{ inputs.rebuild }}
+                build-args: "--target indexer"
+            - name: Build Solr
```

Comment:
Probably would be useful to change this to something like "Build Solr (Sample Index)" rather than have the same name for both

Reply:
Good idea. That has been updated.